src/Cameras/VR/babylon.webVRCamera.ts

```diff
@@ -369,6 +377,7 @@ module BABYLON {
             Quaternion.FromRotationMatrixToRef(this._workingMatrix, this.deviceRotationQuaternion);
 
             super.update();
+            this.updateCalled = false;

```

Comment:
not sure to understand what this boolean is ?

Reply:
to make sure it is called only once. Prevents the infinite loop. It is set to false at the end of this call. Maybe the naming is not the best, but it does the job :)

Reply:
But what is the difference with updateCacheCalled?
I see no declaration for updateCalled

Reply:
Ehhh...
This is obviously not needed. Sorry! I will remove this.